frontend/src/components/layout/AppNavBar/AppNavBar.tsx

```diff
@@ -23,7 +24,9 @@ function AppNavBar() {
             <FaHome size={20} />
           </Nav.Item>
           <AdminDropdown />
-          <SubmitProperty />
+          <FeatureToggle tenant="MOTI" hide>
```

Comment:
Tenant should not be hardcoded.  There is a hook to fetch the value `useTenant()`

Reply:
Well the feature call for "Hide this UI if for MOTI" so yes sadly we need to hardcode the tenant ID there. Happy to discuss other options

Reply:
We don't hardcode this here.  Use the hook.

Reply:
The FeatureToggle component lets us do conditional rendering of the UI based on the tenant but we do need to supply which tenant the logic should work against.

Reply:
I've commented below.  I was misunderstanding.  The value here is just to state this feature is for the specified tenant.